### DISCUSSION TITLE
v2-layout questions and comments - January 2024

### DISCUSSION THREAD

**Discussion by JC5** [firefly-iii/firefly-iii#8360] [General]
Hi there,

Please use this GitHub discussion to report all your Firefly III v2 layout issues and problems, and see if a fix is perhaps already available. Due to the volatile nature of software development, the v2 layout may be buggy or broken. As such the rule is: if it breaks you get to keep both parts. However, I am of course keen to fix these issues so please report them here.

This discussion will be closed and recreated every month, depending on the number of replies and issues etc.

To enable the v2 layout installation in your installation, please set `FIREFLY_III_LAYOUT=v2` in your environment variables. Be aware that pages may be broken, functionality is missing and data loss may occur when you use the layout to submit or change your data.

- [How to enable debug mode](https://docs.firefly-iii.org/how-to/general/debug/)

**Comment by not1q84-1** [comment]
Hi James, 

being the first post, I hope I am using this correctly :-)
I think there is a bug in UI version 2 (v6.1.6) where you cannot add a new tag to an existing transaction, when editing it. 
1. Click on edit for an existing transaction
2. Click on tag entry box. 
3. Start typing - instead of starting to search a red frame and an error icon are appearing; relevant tags are not displayed for selection
![Screenshot 2024-01-13 072236](https://github.com/firefly-iii/firefly-iii/assets/43747205/ba6be84d-b523-4ee6-85ab-e69d5209fc16)

**Reply by JC5** [reply]
Hey, thanks for letting me know! I ran into some troubles with the tag library. I hope to have it fixed soon.

**Comment by brianramseyau** [comment]
Per https://github.com/firefly-iii/firefly-iii/issues/8455

The PWA manifest is not currently being loaded into the new views.
